lib/index.js

```diff
@@ -0,0 +1,127 @@
+'use strict';
+
+const path = require('path');
+const Filter = require('broccoli-persistent-filter');
+const md5Hex = require('md5-hex');
+const extensions = require('markdown-extensions');
+const stringify = require('json-stable-stringify');
+const testGenerators = require('./test-generators');
+const remarkEngine = require('./remark-engine');
+const testGeneratorNames = Object.keys(testGenerators);
+
+function resolveInputDirectory(inputNode) {
+	if (typeof inputNode === 'string') {
+		if (path.isAbsolute(inputNode)) {
+			return inputNode;
+		}
+		return path.join(process.cwd(), inputNode);
+	}
+
+	// eslint-disable-next-line no-underscore-dangle
```

Comment:
Is this necessary? For what I'm reading at the rule docs, it only applies to identifiers, right?

https://eslint.org/docs/rules/no-underscore-dangle